### ISSUE TITLE
Stop using WPF

### ISSUE THREAD

**Issue by nikitawootten** [usnistgov/CloudSecurityArchitectureTool#1]
WPF uses components that are not FIPS-140 approved or allowed. All WPF classes need to be replaced with WinForms classes.

**Comment by spuberman7** [comment]
Hi, 
Would you please elaborate more about your concern? like which level of security of FIPS-140 does CSRC require? level 4?

Do you mean GUI in XAML (XML dialect) is out of date and it needs to be changed to windows form (.cs [design])?

**Comment by nikitawootten** [comment]
Yes, Visual Studio will refuse to build any WPF (GUI in XAML) applications on machines with enforceFIPSPolicy enabled.

Rewriting the project using the older system (Windows Forms) would solve this problem.
